cli/config/configuration.toml.in

```diff
@@ -230,9 +230,23 @@ path = "@NETMONPATH@"
 #   - macvtap
 #     Used when the Container network interface can be bridged using
 #     macvtap.
+#
+#   - none
+#     Used when customize network. Only creates a tap device. No veth pair.
```

Comment:
I think it would be useful to add a link here to the following as that shows users visually more about how this will work:

- https://github.com/kata-containers/documentation/blob/master/architecture.md#networking

Reply:
OK. I will add a PR in documentation repo.